app/javascript/components/search/controls/SearchPanel.js

```diff
@@ -47,7 +47,7 @@ const helpModalContent = (<div>
     target="_blank" rel="noreferrer">wiki
   </a>.  Study authors looking to make their studies more accessible can read our{' '}
```

Comment:
Your solution for spaces before links in JSX is more elegant than mine, so let's remove my redundant solution:

```suggestion
  </a>.  Study authors looking to make their studies more accessible can read our
```